### PR TITLE
Added example "get-users-show" in php language

### PR DESCRIPTION
**Pull request by royopa**

### CODE REVIEW

File: samples/get-users-show/php-twitter/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# GET users/show
+
+* Language: PHP >=5.4.0
+* Dependencies: abraham/twitteroauth
+
+## Install & Build
+
+Make sure you have Composer installed, run `composer install`
+
```
Comment (andypiper) [inline samples/get-users-show/php-twitter/README.md]: Worth saying you also need to create a credentials.txt file and what the contents should be?

Comment (royopa) [inline samples/get-users-show/php-twitter/README.md]: I think this information is already described in the main project README.md
(section Quick Start), or not?

On Thu, Jan 8, 2015 at 12:39 PM, Andy Piper <notifications@github.com>
wrote:

> In samples/get-users-show/php-twitter/README.md
> <https://github.com/twitterdev/api-rosetta-code/pull/2#discussion-diff-22654398>
> :
>
> > @@ -0,0 +1,21 @@
> > +# GET users/show
> > +
> > +* Language: PHP >=5.4.0
> > +* Dependencies: abraham/twitteroauth
> > +
> > +## Install & Build
> > +
> > +Make sure you have Composer installed, run `composer install`
> > +
>
> Worth saying you also need to create a credentials.txt file and what the
> contents should be?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitterdev/api-rosetta-code/pull/2/files#r22654398>.
>



-- 
Rodrigo Prado de Jesus
http://about.me/royopa
royopa@gmail.com

Comment (lfcipriani) [inline samples/get-users-show/php-twitter/README.md]: @andypiper all the samples must load the credentials from credentials.txt file on the root of repository

**Comment by lfcipriani** [comment]
@royopa Thanks for the pull request, can you do the changes above?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
 credentials.txt
+/samples/get-users-show/php-twitter/vendor/
```
Comment (lfcipriani) [inline .gitignore]: Can you add these to a .gitignore file inside the sample dir?

Comment (royopa) [inline .gitignore]: Ok, I will correct this. thanks

**Comment by royopa** [comment]
@lfcipriani I made all changes, Thanks for checking. :-)

**Pull request merged**